### DISCUSSION TITLE
I think `skip_symlinks` should be `follow_symlinks`

### DISCUSSION THREAD

**Discussion by MarcelRobitaille** [abraunegg/onedrive#1903] [General]
I had the same question as #1071. I agree with their final comment. I think it is still not well explained in `USAGE.md`. I think a more descriptive option would be `follow_symlinks`.

- If false, symlinks are ignored (skipped)
- If true, symlinks are followed and the pointed directory structure is uploaded

Alternatively:
- `symlinks = "ignore"`: symlinks are ignored
- `symlinks = "follow"`: follows symlinks and the pointed directory structure is uploaded

At the very least, I think the current behaviour should be clarified in `USAGE.md` and/or the example config. I am willing to create a pull request for this.

**Comment by abraunegg** [comment]
@MarcelRobitaille 
Please can you review: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#handling-symbolic-links

**Reply by MarcelRobitaille** [reply]
I'm sorry. I searched for "symlink", so "symbolic link" didn't come up, leading me to believe that it wasn't documented. I should have read more carefully. I still think `follow_symlinks` is more descriptive of the option.

**Reply by abraunegg** [reply]
@MarcelRobitaille 
> I'm sorry. I searched for "symlink", so "symbolic link" didn't come up, leading me to believe that it wasn't documented. I should have read more carefully. I still think `follow_symlinks` is more descriptive of the option.

The default application behavior is to 'follow' .. the configuration option is to 'skip' .... 

When enabling the option - you are skipping symbolic links from syncing - not enabling to follow.

**Reply by MarcelRobitaille** [reply]
I realize that. I guess my proposal would involve making the default value true. I only make this proposal because it was not clear to me what the default behaviour would be. I see now that it is documented in `USAGE.md`.
